{{cookiecutter.repo_name}}/README.rst

```diff
@@ -0,0 +1,55 @@
+Part of `edX code`__.
+
+__ http://code.edx.org/
+
+{{cookiecutter.project_name}}
+=============================
+
+The ``README.rst`` file, should start with a brief description of the repository
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Overview (please modify)
+------------------------
+
+The ``README.rst`` file should then provide an overview of the code in this
+repository, what the main components are and useful entry points for starting
+to understand the code in more detail.
+
+
+Documentation (please modify)
+-----------------------------
+
+The docs for {{cookiecutter.project_name}} is on Read The Docs:  https://{{cookiecutter.repo_name}}.readthedocs.org
```

Comment:
@srpearce Updated.

@nedbat Yes. I believe that's where @mhoeber (or whoever is assigned to manage docs for the team constructing the new IDA).

I have added an explanation (and retained the existing statement and link to RTD): The ``README.rst`` should include a link to the developer documentation. This documentation should be hosted on
Read the Docs.